### PR TITLE
Update Rust to 1.48.0, fix 'notable trait' display

### LINKED ISSUE #1360: Rust "notable trait" sections broken
Hello,

It seems like last devdocs.io update broke the styles for Rust.
The "notable trait" sections look funny now.
Screenshot from 2020-11-20 14-32-30

Shouldn't that be outside the method signature?

Thanks!
<3

### PR DESCRIPTION
**Pull request by MasterEnoc**
Fixes #1360 

- Modified "notable traits" sections to improve how are displayed in the dom.
- Fixes error in reference/introduction.html while scraping.

### CODE REVIEW

File: lib/docs/filters/rust/clean_html.rb [hunk 1]
```diff
@@ -97,6 +97,22 @@ def call
           node.previous_element.before(node)
         end
 
+        css('.collapse-toggle').remove
+
+        # Fix how notable-traits sections are shown
+
+        css('.method').each do |node|
+          traitSection = node.at_css('.notable-traits')
```
Comment (simon04) [inline lib/docs/filters/rust/clean_html.rb:105]: The handling of `.important-traits` in line 81 seems related. Is this related? Is `.important-traits` still in use?

Comment (MasterEnoc) [reply lib/docs/filters/rust/clean_html.rb:105]: Looks like .important-traits is not in used anymore, I'm going to clean those lines

**Comment by MasterEnoc** [comment]
I updated the version of the scraper and remove unnecessary lines in the clean_html filter.
I tested all the other procedures in the clean_html filter and are still necessary to keep the looking good the style of rust in devdocs, there are a lot of procedures but this filters is applied to three different pages and that is the main reason why is relatively long.
Please proceed to review it.

**Review by simon04 — APPROVED** [review]
Awesome, thank you!

**Pull request merged**